src/main/java/technology/semi/weaviate/client/v1/graphql/query/argument/SortArgument.java

```diff
@@ -0,0 +1,35 @@
+package technology.semi.weaviate.client.v1.graphql.query.argument;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.FieldDefaults;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+@Getter
+@Builder
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class SortArgument implements Argument {
+  String[] path;
+  SortOrder order;
+
+  @Override
+  public String build() {
+    Set<String> arg = new LinkedHashSet<>();
+    if (ArrayUtils.isNotEmpty(path)) {
```

Comment:
what if path is empty? result should be `{path:[] order:asc}` or `{order:asc}`

Reply:
then it will be generated like this: `{path:[] order:asc}`

Reply:
I will change that behaviour in my next PR